internal/controller/operator/instance/dapr_instance_controller_action_apply.go

```diff
@@ -23,13 +26,26 @@ import (
 	"github.com/dapr-sandbox/dapr-kubernetes-operator/pkg/resources"
 )
 
-func NewApplyAction() Action {
-	return &ApplyAction{
+const utoPullPolicySidecarInjector = `
+if (.dapr_sidecar_injector.image | has("name")) and (.dapr_sidecar_injector | has("sidecarImagePullPolicy") | not) 
+then 
+  .dapr_sidecar_injector.sidecarImagePullPolicy = "Always"
+end
```

Comment:
@ItalyPaleAle @salaboy 

Here I've added a sort of customizer that automatically set the sidecar pull policy in case of a custom image is set.  

I think we should make any customization an opt-in (and likely track it in the status sub resource), what do you think ?